medusa/server/api/v2/config.py

```diff
@@ -448,7 +450,8 @@ def sections(cls):
         """Get the available section names."""
         return [
             name[5:]
-            for (name, function) in inspect.getmembers(cls, predicate=inspect.isfunction)
+            for (name, function)
+            in inspect.getmembers(cls, predicate=lambda f: inspect.isfunction(f) or inspect.ismethod(f))
```

Comment:
Iirc you can pass multiple predicates as a list, so you can avoid the lambda.

Reply:
```py
Python 3.7.3 (v3.7.3:ef4ec6ed12, Mar 25 2019, 22:22:05) [MSC v.1916 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from medusa.server.api.v2.config import DataGenerator
>>> import inspect
>>> inspect.getmembers(DataGenerator, predicate=[inspect.isfunction, inspect.ismethod])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Python\Python37\lib\inspect.py", line 354, in getmembers
    if not predicate or predicate(value):
TypeError: 'list' object is not callable
>>>
```